### PR TITLE
Added Nim Game

### LINKED ISSUE #713: Nim Game Problem in C++,Java, Python
Please assign it to me.

### PR DESCRIPTION
**Pull request by MastersAbh**
Fixes #713 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added implementation of Nim game in C++, Java and Python.

#### Languages Used:
- C++
- Java
- Python

#### Files Added:
- Nim_Game.cpp
- Nim_Game.java
- Nim_Game.py

### CODE REVIEW

**Comment by jainaman224** [comment]
The PR template is incomplete. Please complete it.

File: Nim_Game/Nim_Game.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+piles = [3,4,1,7,6]
+print("Player",calculateWinner(starting_player, piles, no_of_piles),"will win.")
+
+
+'''
+Input : starting_player = 0,
+        piles = {3,4,1,7,6}
+Output: Player 0 will win.
+'''
+
```
Comment (jainaman224) [inline Nim_Game/Nim_Game.py]: Add only one new line at the end of file.

**Comment by MastersAbh** [comment]
Please review @jainaman224

**Comment by jainaman224** [comment]
Great PR.

**Pull request merged**